src/NServiceBus.AcceptanceTests/Core/SelfVerification/EndpointNameEnforcementTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace NServiceBus.AcceptanceTests.Core.SelfVerification
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+    using AcceptanceTesting;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class EndpointNameEnforcementTests : NServiceBusAcceptanceTest
```

Comment:
Sidenote: Since this is just an enforcement test and not a "real" acceptance test I don't think deriving from `NServiceBusAcceptanceTest` is strictly needed

Reply:
It is because the test that checks for it fails otherwise. 😄

Reply:
I was able to run this test manually w/o extending the base class... Not that it matters, just weird that this test has to have a base class 🙂

Reply:
Right, but TC failed because it didn't have it, which is why I requested it be added.